### ISSUE TITLE
Error when connecting to Aurora

### ISSUE THREAD

**Issue by pquadri** [duckdb/postgres_scanner#129]
### What happens?

After https://github.com/Mytherin/postgres_scanner/commit/c37606d0f98cbda64273ba220f0de344537c8cdf connecting to an Aurora results in 
`ERROR:  Function pg_stat_get_wal_receiver() is currently not supported in Aurora`

Can we make that bit optional?

### To Reproduce

try to run a postgres_scan from aurora, i have issues with postgres_attach as well but that might be on my connection.
does not happen with the same query on duckdb 0.9.0 (which i assume is old enough to not have that line)

### OS:

mac os x

### PostgreSQL Version:

14

### DuckDB Version:

9.2.0

### DuckDB Client:

Python

### Full Name:

Paolo Quadri

### Affiliation:

Remote.com

### Have you tried this on the latest `main` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Mytherin** [comment]
Should be fixed in #131

**Comment by Mytherin** [comment]
You are most likely still using the old version of the extension, try installing the nightly:

```sql
force install postgres_scanner from 'http://nightly-extensions.duckdb.org';
```
